arduino/smartcar/smartcar.ino

```diff
@@ -1,9 +1,39 @@
 #include <Smartcar.h>
 
-void setup() {
-  // put your setup code here, to run once:
+ArduinoRuntime arduinoRuntime;
+BrushedMotor leftMotor(arduinoRuntime, smartcarlib::pins::v2::leftMotorPins);
+BrushedMotor rightMotor(arduinoRuntime, smartcarlib::pins::v2::rightMotorPins);
+DifferentialControl control(leftMotor, rightMotor);
+
+GY50 gyroscope(arduinoRuntime, 37);
+const auto pulsesPerMeter = 600;
+DirectionlessOdometer leftOdometer{ arduinoRuntime,smartcarlib::pins::v2::leftOdometerPin,[]() { leftOdometer.update(); },pulsesPerMeter };
+DirectionlessOdometer rightOdometer{ arduinoRuntime,smartcarlib::pins::v2::rightOdometerPin,[]() { rightOdometer.update(); },pulsesPerMeter };
+SmartCar car(arduinoRuntime, control, gyroscope, leftOdometer,rightOdometer);
+ 
+const int triggerPin           = 6;  //D6
+const int echoPin              = 7;  //D7
+const unsigned int maxDistance = 100;
+SR04 front{arduinoRuntime, triggerPin, echoPin, maxDistance};
```

Comment:
You make claims about `1.5` meters later, yet the max distance you have set the sensor to detect is 1 meter.